test/jani_generator/_test_data/uc2_assembly/Plugins/IsBlockFell.scxml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would suggest to call this skill:
- IsBlockDown or
- HasBlockFallen

or something else that makes sense grammatically ;)

Reply:
This is also something to ask @m-morelli